packages/seed/src/cli.ts

```diff
@@ -596,6 +596,12 @@ function addGenerateCommands(cli: Argv) {
                                     "Path to write the changelog to, if not provided, will write to cwd. Note this should be a directory, not a filename.",
                                 string: true,
                                 demandOption: false
+                            })
+                            .option("clean-directory", {
```

Comment:
Not sure what the _right_ name is here, but it's less important because this is scoped to `seed`.

Reply:
Yeah totally, `overwrite` felt incorrect, etc
If we want to rename it down the line we can for sure since this isn't meant to be user facing